### DISCUSSION TITLE
post install

### DISCUSSION THREAD

**Discussion by paijho** [stevedat/HP-Elitebook-830-G6-Hackintosh#22] [General]
may you share post install efi? i have trouble with monterey. paijho@gmail.com

**Comment by mrmtmfmf** [comment]
I have successfully installed Catalina 10.15.7 on my hp elitebook 830 g6 using your published EFI. Thank you! However, I have problems with Wi-Fi and touchpad operation. Can you please share me a post-install EFI to fix these?
hackintosh_mrmtmfmf@fuwamofu.com

**Reply by stevedat** [reply]
I no longer use it, so not sure my folder could help.
